### ISSUE TITLE
No way to test landing pages

### ISSUE THREAD

**Issue by bandrel** [gophish/gophish#114]
No way to test landing pages without creating new campaign. 

Test email sends URL, but you get a 404 error when going to the phish URL.

**Comment by jordan-wright** [comment]
Hey @bandrel,

Thanks for the report! Yes, unfortunately there's no current way to test landing pages. This deals with the way we dynamically generate landing pages based on the result ID. I've considered a good way to deal with this, but haven't come up with a solution yet.

I'll assign this to 0.2, and we'll keep this issue open to track how things are going.

**Comment by jcamil** [comment]
I have noticed that if one edits the landing page html, and clicks on the "source" button, the landing page is at least partially rendered.

**Comment by jordan-wright** [comment]
Absolutely! That's one of the coolest parts of the editor. In fact, when having the source mode enabled, you can actually click in the pane and edit the content WYSWYG style!

The same applies for email templates. And there's a full screen button if you want to see it full screen!

**Comment by netektives** [comment]
What about a semi-static result ID that just doesn't get logged?  Have a link on the import page that will open up the landing page in separate tab?

**Comment by bandrel** [comment]
I agree on the semi-static method.  The wysiwyg editor just renders the html and does not verify that the webserver is running etc.

**Comment by acklenx** [comment]
Is this being worked on?  I'm really surprised that there is no way to test is the webserver is working (particularly with ssl).  You can view the page formating, but then you have to actually start a new campaign just to test.

+1

**Comment by acklenx** [comment]
Woohoo!  Thanks for looking at this
